### PR TITLE
Integrate with Entity Browser for selection

### PR DESCRIPTION
**Pull request by slashrsm**
https://www.drupal.org/node/2510596

### CODE REVIEW

File: src/Form/EntityEmbedDialog.php [hunk 1]
```diff
@@ -159,16 +185,40 @@ public function buildSelectStep(array &$form, FormStateInterface $form_state) {
...
+      $form['attributes']['entity_browser']['#theme_wrappers'] = ['container'];
+      $form['attributes']['entity_browser']['browser'] = $entity_browser->getDisplay()->displayEntityBrowser();
+      $form['attributes']['entity_browser']['entity-id'] = [
+        '#type' => 'hidden',
+        '#default_value' => $entity ? $entity->id() : '',
+        '#attributes' => ['class' => ['eb-target']]
+      ];
+      $form['#attached']['library'][] = 'entity_browser/common';
+      $form['#attached']['drupalSettings']['entity_browser'] = [
+        'field_settings' => [$entity_browser->getDisplay()->getUuid() => ['cardinality' => 1]]
```
Comment (slashrsm) [inline src/Form/EntityEmbedDialog.php:203]: This configuration value was moved to a different part of the settings array. See https://github.com/drupal-media/entity_browser/commit/e712253121cb3fef3abf1909da1e2bfd3bc35938

File: src/Form/EntityEmbedDialog.php [hunk 2]
```diff
@@ -571,4 +625,15 @@ public function getDisplayPluginOptions(EmbedButtonInterface $embed_button, Enti
     natsort($plugins);
     return $plugins;
   }
+
+  /**
+   * Registers JS callback that gets entities from entity browser and updates
+   * form values accordingly.
+   */
+  public function registerJSCallback(RegisterJSCallbacks $event) {
```
Comment (slashrsm) [inline src/Form/EntityEmbedDialog.php:633]: This is not needed any more. Can be removed. See https://github.com/drupal-media/entity_browser/commit/a2fddbbe467049726aca9d4b985f3f3a83c9552d

**Comment by webflo** [comment]
New PR in https://github.com/drupal-media/entity_embed/pull/192

**Comment by slashrsm** [comment]
Was merged as part of #192.

**Pull request closed without merging**